src/components/OurOffer/OfferCard/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Box, Typography, Button } from "@mui/material";
+import classnames from "classnames";
+import CheckIcon from "@assets/icon/check-icon.svg?react";
+import StarIcon from "@assets/icon/star-icon.svg?react";
+import { IPlanProps, ITextProps } from "../helpers";
+import PlanPrice from "./PlanPrice";
+import useStyles from "./styles";
+
+interface OfferCardProps {
+  plan: IPlanProps;
+  text: ITextProps;
+}
+
+const OfferCard: React.FC<OfferCardProps> = ({ plan, text }) => {
+  const { classes } = useStyles();
+
+  function handleFeatureList() {
+    return plan.features.map((feature, index) => (
+      <li
+        key={index}
+        className={classnames({
+          [classes.featureItem]: true,
```

Comment:
You could do it like this:

`classnames(classes.featureItem, { ... })`

and also use a ternary:

```
classnames(
  classes.featureItem,
  feature.includes(text.benefitText) ? classes.titleFeature : classes.textFeature
)
```

Looks a bit more elegant to me. Whatever you prefer. 🙂